apps/sms/views/conversation/test/unit/conversation_test.js

```diff
@@ -333,7 +333,9 @@ suite('conversation.js >', function() {
     suite('rendering suggestions list', function() {
       var suggestionRenderer, unknownRenderer;
       var contact, unknown;
+      var clock;
       setup(function(done) {
+        clock = sinon.useFakeTimers(Date.now());
```

Comment:
I really don't think you'll need to give the Date.now() here since it's not absolute timestamp sensitive test case. Since the  sinon.useFakeTimers already set at the top of the setup, you can simply use this.sinon.clock.tick(ms) without further setup for your test(and you don't have to restore unless you want to do something specific in the same test)